packages/itwin/grouping-mapping-widget/src/widget/components/GroupQueryBuilderContainer.tsx

```diff
@@ -10,25 +10,35 @@ import type {
 } from "@itwin/presentation-frontend";
 import { KeySet } from "@itwin/presentation-common";
 import { GroupQueryBuilderApi } from "../../api/GroupQueryBuilderApi";
-import { PropertyGridWrapperApp } from "./property-grid/PropertyGridWrapper";
-import { GroupQueryBuilderContext } from "./context/GroupQueryBuilderContext";
+import { PropertyGridWrapper } from "./property-grid/PropertyGridWrapper";
+import { PropertyGridWrapperContext as PropertyGridWrapperContext } from "./context/PropertyGridWrapperContext";
```

Comment:
Import alias is the same as the imported class.

Reply:
Removed. Probably was an artefact from renaming